promotions/spec/models/solidus_promotions/order_adjuster_spec.rb

```diff
@@ -67,6 +67,7 @@
         it " will not create the adjustment" do
           expect {
             subject
+            order.save!
```

Comment:
We only want to check for amount zero and marked for destruction here and not save the order.